### ISSUE TITLE
Fusion installer does not upgrade the asset root on RHEL 7

### ISSUE THREAD

**Issue by tst-lsavoie** [google/earthenterprise#444]
On RHEL 7, if you upgrade GEE from 5.1.3 to 5.2.0, the asset root is not upgrade when you run install_fusion.sh.

The workaround is to upgrade the asset root manually by running the command that is printed when you try to start the fusion service.

**Comment by tst-ccamp** [comment]
This wasn't caused by setting a non-default asset root, like /usr/google/gevol, was it?

**Comment by tst-lsavoie** [comment]
It could have been.  This was on a system where I was running run_tutorial.sh, and that uses a non-default asset root.

**Comment by tst-ccamp** [comment]
Yeah, I have a feeling that this just fails for non-default asset roots, no matter the OS. I'm also not entirely sure if it's possible to detect that a non-default asset root has been created. But, that's something we can research.

**Comment by tst-ppenev** [comment]
Well, they asset roots are stored in a configuration file, aren't they? It should be possible to read configuration files.
